_episodes/12.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: episode
+title:  "12: Trino gets super visual with Superset!"
+date: 2021-03-04
+tags: trino visualization superset apache client jdbc python
+youtube_id: "0NHUs-TERtk"
+wistia_id: "ggayk0tihh"
+sections: 
+   - title: "Concept of the week"
+     desc: "Trino client, Python, and Apache Superset "
+     time: 463
+   - title: "PR of the week"
+     desc: "Superset PR 13105 feat: first step native support Trino"
+     time: 1076
+   - title: "PR Demo"
+     desc: "PR Demo: Superset PR 13105 feat: first step native support Trino"
```

Comment:
Demonstration of Apache Superset querying Trino to create a number of visualizations

Reply:
Also in section header below..

Reply:
This may get long in the summaries but how about.

Demo of Superset querying Trino to create visualizations